stdlib/universe/window_interval_2.go

```diff
@@ -0,0 +1,287 @@
+package universe
+
+import (
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/internal/errors"
+	"github.com/influxdata/flux/interval"
+	"github.com/influxdata/flux/plan"
+	"github.com/influxdata/flux/runtime"
+	"github.com/influxdata/flux/values"
+)
+
+const WindowIntervalKind2 = "windowInterval2"
+const WindowKind2 = "window2"
+const DefaultStartColumn = "_start"
+const DefaultStopColumn = "_stop"
+const DefaultTimeColumn = "_time"
+
+type WindowIntervalOpSpec2 struct {
+	Every       flux.Duration `json:"every"`
+	Period      flux.Duration `json:"period"`
+	Offset      flux.Duration `json:"offset"`
+	CreateEmpty bool          `json:"createEmpty"`
+}
+
+func init() {
+	windowSignature := runtime.MustLookupBuiltinType("universe", "window2")
+	runtime.RegisterPackageValue("universe", WindowKind2, flux.MustValue(flux.FunctionValue(WindowKind2, createWindowOpSpec, windowSignature)))
```

Comment:
This file contains three types:

An OpSpec, ProcedureSpec and a Transformation, all of those are the same as the window_interval.go code. 

I think you can delete all of those from this file and it will still work. All that is needed is that you register the package value as you do on this line here. Then under the hood they become the same function.

Reply:
I double checked, but I could be wrong, but I think those three functions are actually different now that we don't include the time, start, and stop columns.